unity-renderer/Assets/Builder/Scripts/Gizmos/DCLBuilderScaleGizmo.cs

```diff
@@ -22,12 +24,38 @@ public override void Initialize(Camera camera, Transform cameraTransform)
         public override void SetSnapFactor(DCLBuilderGizmoManager.SnapInfo snapInfo) { snapFactor = snapInfo.scale; }
 
         public override float TransformEntity(Transform entityTransform, DCLBuilderGizmoAxis axis, float axisValue)
+        {
+            // In order to avoid to make the scale of each selected entity dependent of the 'entityTransform' parent,
+            // we temporally move all entities to the local position (0,0,0) before calculate the new scale.
+            foreach (Transform entity in entityTransform)
+            {
+                entitiesOriginalPositions.Add(entity, entity.transform.position);
+                entity.transform.localPosition = Vector3.zero;
```

Comment:
this should be `position` instead of `localPosition` right? let's use the same position term to be consistent, even though probably in this case `poition` and `localPosition` do the same thing if the entity doesn't have a parent GO

Reply:
But in this case I want to set its local position, not the world one, because the entity has a parent and I want to place temporally the child to the (0,0,0) position relative to the parent.